app/Jobs/PushServerUpdateJob.php

```
@@ -402,66 +403,59 @@ private function updateApplicationPreviewStatus(string $applicationId, string $p
+            return;
         }
+
+        // Batch update: mark all not-found applications as exited (excluding already exited ones)
+        Application::whereIn('id', $notFoundApplicationIds)
+            ->where('status', 'not like', 'exited%')
+            ->update(['status' => 'exited']);
     }
 
     private function updateNotFoundApplicationPreviewStatus()
     {
         $notFoundApplicationPreviewsIds = $this->allApplicationPreviewsIds->diff($this->foundApplicationPreviewsIds);
```

PHPMD (2.15.0)

Avoid excessively long variable names like $notFoundApplicationPreviewsIds. Keep variable name length under 20. (undefined)

(LongVariable)